### PR TITLE
Maintenance: release into develop

### PR DESCRIPTION
**Pull request by edburyenegren-okta**

### CODE REVIEW

**Review by arnoldsandoval-okta — APPROVED** [review]

File: packages/odyssey/src/scss/base/_typography-list.scss [hunk 1]
```diff
@@ -50,6 +50,11 @@ dl:not([class]) {
   max-width: $max-line-length;
   margin: 0 0 $spacing-s 0;
   padding: 0;
+  font-size: $size-body-sentence;
+
+  &:last-child {
+    margin-bottom: 0;
+  }
```
Comment (santhoshbalakrishnan) [inline packages/odyssey/src/scss/base/_typography-list.scss:57]: How did this come in here?

**Review by santhoshbalakrishnan — APPROVED** [review]
Change looks good, though highly recommend automating the version updates, since there are multiple places where this is happening.

File: packages/odyssey/src/scss/base/_typography-list.scss (hunk 1 above)
Comment (edburyenegren-okta) [reply packages/odyssey/src/scss/base/_typography-list.scss:57]: This was pushed to the release branch as a UI fix for the PR - Design had signed off on something different than what got shipped.

**Comment by edburyenegren-okta** [comment]
@santhoshbalakrishnan the current versioning process was requested by UI Core as laid out in the README - happy to swap to something less custom.

**Pull request merged**